README.md

```diff
@@ -369,6 +369,10 @@ module. On Windows, this can be a UNC path to the MSI.  Defaults to undef.
 This variable is passed to the package resources' *install_options* parameter.
 Defaults to the value in ::splunk::params.
 
+#### `manage_splunk_user`
+
+Whether or not to manage the user splunk runuser.  Defaults to false.
```

Comment:
Is the extra "user" before "splunk" a mistake? The phrase reads oddly and you didn't type it this way in the actual parameter documentation in the manifest